src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md

```diff
@@ -47,9 +47,7 @@ So, with that, we logged the metric to the `EventCounter`, but unless we can act
 
 There is an extra keyword that you will need to specify the turn on the EventCounters.
 
-```
-PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect
-```
+<code>PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect</code>
```

Comment:
Ah, I see - the bold in the middle is not bold. It is actually syntax of the command. So we want intentionally to have triple backtick.